ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -67,6 +68,8 @@ public NonBlockingRouterMetrics(MetricRegistry metricRegistry) {
     slippedPutSuccessCount = metricRegistry.counter(MetricRegistry.name(PutManager.class, "-SlippedPutSuccessCount"));
     putBlobErrorCount = metricRegistry.counter(MetricRegistry.name(PutManager.class, "-PutBlobErrorCount"));
     operationAbortCount = metricRegistry.counter(MetricRegistry.name(NonBlockingRouter.class, "-OperationAbortCount"));
+    unexpectedErrorCount =
```

Comment:
I see that there are metrics for this in the metrics framework PR.  I will integrate those changes once that PR is merged.